src/python_testing/TC_EPREF_2_1.py

```diff
@@ -219,7 +220,7 @@ async def test_TC_EPREF_2_1(self):
                               )
 
         else:
-            logging.info("Device does not support EnergyBalance feature and related attributes, skipped Test Step 2 to 5")
+            logging.info("Device does not support EnergyBalance feature and related attributes, skipped Test Step 3 to 5")
```

Comment:
The test script will report failure if not all test returned in `steps_TC_EPREF_2_1` were marked as stepped. I think that you should add something like that if test are skipped:
```python
self.skip_step('3')
self.skip_step('4')
self.skip_step('4a')
self.skip_step('4b')
self.skip_step('5')
```
or
```python
for step in ['3', '4', '4a', '4b', '5']:
   self.skip_step(step)
```

The same for `feature_map` without `kLowPowerModeSensitivity`.